_docs/events/completed-checkout-step.md

```diff
@@ -47,5 +47,5 @@ if (checkoutSteps.indexOf(checkoutStep) !== -1) {
 };
 ```
 
-#### Required by the following integrations:
+#### Optional for the following integrations:
```

Comment:
Completed checkout step is required for enhanced eCommerce. Otherwise, you will not have enhanced e-commerce.

Also, the word "optional" is misleading. If you would like to add additional section it would be better to say "Used in the following integrations"